UT4MasterServer/Controllers/UnrealTournamentMatchmakingController.cs

```diff
@@ -255,18 +255,20 @@ public async Task<IActionResult> ListGameServers([FromBody] GameServerFilter fil
 
 		var servers = await matchmakingService.ListAsync(filter);
 
-		//var list = new GameServer[]
-		//{
-		//	new GameServer("we", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("cracked", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("the", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("code", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("and", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("entered", "[DS]dallastn-22938", "192.223.24.243"),
-		//	new GameServer("the", "[DS]dallastn-22938", "192.223.24.243"), // does not show, due to duplicate data
-		//	new GameServer("matrix", "[DS]dallastn-22938", "192.223.24.243"),
-		//};
-
+#if DEBUG
```

Comment:
If you leave this uncommented, then please add `//#define SHOW_TEST_HUBS` (notice comment) at the top.
Then change this line (258) to `#if DEBUG && SHOW_TEST_HUBS`

Then uncomment the `#define` when you want to test locally.

Reply:
I'm just gonna revert it, the test data doesn't really work anyway. For testing I just returned the response from the prod server manually. Should probably come up with a better means of getting test data at some point.